idb/postgres/internal/encoding/encode.go

```diff
@@ -53,7 +54,39 @@ func convertEvalDelta(evalDelta types.EvalDelta) types.EvalDelta {
 	return evalDelta
 }
 
+func desanitizeNull(str string) string {
+	return strings.ReplaceAll(str, `\u0000`, "\x00")
+}
+
+// SanitizeNullForQuery converts a string into something postgres can store in a jsonb column.
+func SanitizeNullForQuery(str string) string {
+	return strings.ReplaceAll(str, "\x00", `\\u0000`)
+}
+
+// sanitizeNull converts a string into something postgres can store in a jsonb column.
+func sanitizeNull(str string) string {
+	return strings.ReplaceAll(str, "\x00", `\u0000`)
+}
+
+// SanitizeParams sanitizes all AssetParams that need it.
+func SanitizeParams(params types.AssetParams) types.AssetParams {
```

Comment:
The convention I had in mind is: functions like `EncodeSignedTxnWithAD()` which calls `convertSignedTxnWithAD()` and does json encoding in this file, and the code outside that only uses `EncodeSignedTxnWithAD()`. So, `SanitizeParams()` should be renamed to `convertAssetParams()` and we need a function called `EncodeAssetParams()`.

The decoding functions should be in `decode.go` instead, but I think we shouldn't have them. They should have a similar convention.

Reply:
I think this suggestion breaks things down more than is useful. I'll make some updates to fix the naming conventions.